src/main/scala/it/pagopa/interop/purposeprocess/service/impl/RiskAnalysisServiceImpl.scala

```diff
@@ -0,0 +1,39 @@
+package it.pagopa.interop.purposeprocess.service.impl
+
+import it.pagopa.interop.purposeprocess.model.riskAnalysisTemplate.RiskAnalysisFormConfig
+import it.pagopa.interop.tenantmanagement.client.model.TenantKind
+import it.pagopa.interop.tenantmanagement.client.model.TenantKind.{GSP, PA, PRIVATE}
+import it.pagopa.interop.purposeprocess.service.RiskAnalysisService
+
+import scala.io.Source
+import spray.json._
+
+object RiskAnalysisServiceImpl extends RiskAnalysisService {
```

Comment:
which is the benefit of this additional object?

Reply:
Nothing. But it is more beautiful to see.